entries/s/stanford.edu.json

```diff
@@ -1,12 +1,12 @@
 {
   "Stanford University": {
     "domain": "stanford.edu",
-    "url": "https://www.stanford.edu/",
     "tfa": [
       "sms",
       "call",
       "custom-software",
-      "custom-hardware"
+      "custom-hardware",
```

Comment:
Looks like the Fortitoken option is deprecated so it seems wise to remove it from the list.
```suggestion
```

Reply:
Where does it say deprecated? [All I see](https://uit.stanford.edu/service/authentication/twostep/token) is that they recommend security keys over it.

Reply:
That was what I concluded from the message but you're right, it's not spelled out, so either remove the custom-hardware method or add `Fortitoken` as a custom-hardware element.

Reply:
Sure thing